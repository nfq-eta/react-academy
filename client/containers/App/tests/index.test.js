import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import {AppDisconnected as App} from '../index';

describe('App', () => {
  it('match a snapshot', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
