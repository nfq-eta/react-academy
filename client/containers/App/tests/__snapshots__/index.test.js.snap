// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App match a snapshot 1`] = `
<div
  className="app-image"
  style={
    Object {
      "backgroundImage": "url('https://image.tmdb.org/t/p/original/AdYJMNhcXVeqjRenSHP88oaLCaC.jpg')",
    }
  }
>
  <div
    className="app-colors"
  >
    <div
      className="app-container"
    >
      <div
        className="header"
      >
        <Link
          className="header__logo"
          replace={false}
          to="/"
        >
          MOVIES
        </Link>
        <div
          className="header__action"
        >
          Genres
        </div>
        <div
          className="header__action is-active"
        >
          Most popular
        </div>
        <div
          className="header__action"
        >
          Top rated
        </div>
        <div
          className="header__action"
        >
          Upcomming
        </div>
        <div
          className="header-search"
        >
          <input
            className="header-search__input"
            type="text"
          />
          <div
            className="header-search__button"
          >
            <i
              className="fa fa-search"
            />
          </div>
        </div>
      </div>
      <div
        className="container-fluid"
      >
        <Home />
      </div>
    </div>
  </div>
</div>
`;
